pkg/datapath/linux/config/config_test.go

```diff
@@ -117,11 +119,46 @@ func writeConfig(t *testing.T, header string, write writeFn) {
 	}
 }
 
+func setupCiliumDummyDevices(t *testing.T, ns *netns.NetNS) {
+	t.Helper()
+	require.NoError(t, ns.Do(func() error {
+		ciliumNet := &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: "cilium_net"}}
+		if err := netlink.LinkAdd(ciliumNet); err != nil {
+			return err
+		}
+		if err := netlink.LinkSetUp(ciliumNet); err != nil {
+			return err
+		}
+		ciliumHost := &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: "cilium_host"}}
+		if err := netlink.LinkAdd(ciliumHost); err != nil {
+			return err
+		}
+		if err := netlink.LinkSetUp(ciliumHost); err != nil {
+			return err
+		}
+		return nil
+	}))
+
+	t.Cleanup(func() {
+		_ = ns.Do(func() error {
+			netlink.LinkDel(&netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: "cilium_net"}})
+			netlink.LinkDel(&netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: "cilium_host"}})
+			return nil
+		})
+	})
+}
+
 func TestWriteNodeConfig(t *testing.T) {
-	setupConfigSuite(t)
```

Comment:
`testutils.PrivilegedTest(t)` is still necessary here. (setupConfigSuite calls it)